website/server/models/user/methods.js

```diff
@@ -20,15 +20,16 @@ schema.methods.getGroups = function getUserGroups () {
   return userGroups;
 };
 
-schema.methods.sendMessage = async function sendMessage (userToReceiveMessage, message) {
+schema.methods.sendMessage = async function sendMessage (userToReceiveMessage, options) {
```

Comment:
Do you use any particular format for these, like with the public API or should I just throw an ad-hoc comment above?